src/containers/StructurePage/StructureContainer.tsx

```diff
@@ -149,9 +205,12 @@ const StructureContainer = ({
           {messageBox && <Column>{messageBox}</Column>}
           <Column colEnd={7}>
             <StructureBanner
-              onChange={toggleShowFavorites}
-              checked={showFavorites}
+              setShowFavorites={toggleShowFavorites}
+              showFavorites={showFavorites}
+              setShowLmaSubjects={toggleShowLmaSubjects}
+              showLmaSubjects={showLmaSubjects}
               nodeType={rootNodeType}
+              hasLmaSubjects={!!lmaSubjectsQuery.data?.length}
```

Comment:
Toggelen skal skjules når man ikke har LMA-fag. Gikk også for at toggelen skjules ved loading/error, tar gjerne innspill på det hvis noen har!